tensorflow/compiler/xla/service/hlo_schedule.cc

```diff
@@ -261,8 +261,8 @@ Status HloSchedule::Verify() const {
           << " appears more than once in the schedule";
       pos++;
     }
-
-    TF_RET_CHECK(instruction_position.size() ==
+    const int64 instruction_position_size = instruction_position.size();
+    TF_RET_CHECK(instruction_position_size ==
```

Comment:
```suggestion
    const int64 num_instructions = instruction_position.size();
    TF_RET_CHECK(num_instructions ==
```
Then merge with next line and also change in the error message below.